src/components/Body/Body.js

```diff
@@ -41,7 +41,7 @@ export default function Body() {
         {componentToRender}
       </Route>
       <Route exact path="/:foo+">
-        <DefaultRedirect />
+        <PageNotFoundScreen />
```

Comment:
I am surprised this is working. However, to be more robust and clear, change this to:
`<Route component={PageNotFoundScreen} />`

Reply:
Done.